zebra-chain/src/parameters/network.rs

```diff
@@ -42,6 +44,14 @@ impl Network {
             Network::Testnet => 18233,
         }
     }
+
+    /// Get the minimum mandatory checkpoint for this network.
+    pub fn mandatory_checkpoint_height(&self) -> Height {
+        // Currently this is Canopy for both networks.
```

Comment:
Let's expand the docs a bit here, because it's the first place people will look when they see this method:
```suggestion
    /// Get the mandatory minimum checkpoint height for this network.
    ///
    /// Mandatory checkpoints are a Zebra-specific feature.
    /// If a Zcash consensus rule only applies before the mandatory checkpoint,
    /// Zebra can skip validation of that rule.
    pub fn mandatory_checkpoint_height(&self) -> Height {
        // Currently this is the Canopy activation height for both networks.
```